src/Tribe/Assets.php

```diff
@@ -18,7 +18,7 @@ public function enqueue_scripts() {
 				[ 'event-tickets-reset-css', 'reset.css' ],
 				[ 'event-tickets-tickets-css', 'tickets.css', [ 'dashicons', 'event-tickets-reset-css' ] ],
 				[ 'event-tickets-tickets-rsvp-css', 'rsvp.css', [] ],
-				[ 'event-tickets-tickets-rsvp-js', 'rsvp.js', [ 'jquery', 'jquery-ui-datepicker' ] ],
+				[ 'event-tickets-tickets-rsvp-js', 'rsvp.js', [ 'jquery' ] ],
```

Comment:
* :no_entry_sign: Short array syntax is not allowed
* :no_entry_sign: Short array syntax is not allowed

<sub>hash: e7a50a8867246744676a1d24d73f85db</sub>